main_js.go

```diff
@@ -711,6 +713,18 @@ func (pe *pcdeditor) runImpl(ctx context.Context) error {
 			pointSize := pe.cmd.PointSize()
 			selectMode := pe.cmd.SelectMode()
 
+			totalPoints := 0
+			if hasPointCloud && pp.Points > 0 {
+				totalPoints += pp.Points
+			}
+			if hasSubPointCloud && ppSub.Points > 0 && selectMode == selectModeInsert {
+				totalPoints += ppSub.Points
+			}
```

Comment:
We could move this inside the `if pe.vi.dragging()` block as `totalPoints` seems to be used only there.